consensus/ibft/src/test/java/tech/pegasys/pantheon/consensus/ibft/validation/NewRoundMessageValidatorTest.java

```diff
@@ -289,4 +268,25 @@ public void lastestPreparedCertificateMatchesNewRoundProposalIsSuccessful() {
 
     assertThat(validator.validateNewRoundMessage(msg)).isTrue();
```

Comment:
Based on the name for the this test and the added comments referring to failing validation, should this assertion be a `isFalse()` rather than `isTrue()`?

Reply:
It looks like this test is proving the successful case when we have multiple PrepareCertificates, so ATM, I think its correct.

Reply:
My bad, in GitHub interface I'd assumed choosing to expand the collapsed section fully expanded ...turns out multiple clicks were needed, the `isTrue()` is in the succeed test, not the fail test.